### DISCUSSION TITLE
brew update...and nothing but a blinking cursor on a blank line

### DISCUSSION THREAD

**Discussion by niemand0** [Homebrew/discussions#2869] [Getting started]
The directory `/usr/local/var/homebrew` has a date of July 6, 2017. So it's been a few years since I last used brew. Now that I have run `brew update`, it looks to be just sitting there. Can I CTRL-C out of this without breaking anything? And then what do I do?
I'm on macOS Catalina 10.15.7.
![image](https://user-images.githubusercontent.com/1043792/151513411-3adee444-9021-440c-97e8-6440f056d6d1.png)

**Comment by SMillerDev** [comment]
Yeah, doing ctrl-c should be fine. Using `brew update-reset` might be the best solution

**Reply by niemand0** [reply]
Thanks for the reply. After reading your message, I went back to the terminal window and saw, to my surprise, about 2-feet worth of scrolling:
![image](https://user-images.githubusercontent.com/1043792/151519024-68bda7aa-e934-43b2-8969-783d1da425c3.png)
![image](https://user-images.githubusercontent.com/1043792/151519152-7d0ad9da-708f-4522-803f-d077f986765b.png)
It would help alleviate anxiety if brew output some kind of message indicating what to expect (like "Expect a blank screen for 30 to 40 minutes"). I guess I can skip `brew update-reset`.

**Reply by carlocab** [reply]
The typical `brew update` doesn't take anywhere near that long -- and `brew` currently doesn't really have a good way of finding out how long the update will take either. You could probably run `brew update --verbose --debug` to get more feedback from the update, though.
